docs/project-model/overview.md

```diff
@@ -0,0 +1,8 @@
+# .Net core project system
+
+Since .net core can be used on multiple platform it cannot rely on a single editor or enviroment. Because of this, and to prevent every editor to make their own implementation, the project system for .net core is now part of the platform.
```

Comment:
I really ment this section to just be a placeholder, but maybe its better to remove it completely for now until the churn is over?